projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.spec.ts

```diff
@@ -488,138 +476,85 @@ describe('Integration Tests: WizardComponent with Mock Parent', () => {
 
```

Comment:
it the line above again not an implementation detail?

Reply:
Done